invokeai/app/services/model_install/model_install_default.py

```diff
@@ -86,67 +114,28 @@ def event_bus(self) -> Optional[EventServiceBase]:  # noqa D102
 
     def start(self, *args: Any, **kwarg: Any) -> None:
         """Start the installer thread."""
-        self._start_installer_thread()
-        self.sync_to_config()
+        with self._lock:
+            if self._running:
+                raise Exception("Attempt to start the installer service twice")
+            self._start_installer_thread()
+            self._remove_dangling_install_dirs()
+            self.sync_to_config()
 
     def stop(self, *args: Any, **kwarg: Any) -> None:
         """Stop the installer thread; after this the object can be deleted and garbage collected."""
-        self._install_queue.put(STOP_JOB)
```

Comment:
Both `start` and `stop` are called with a single `invoker: Invoker` arg.

Reply:
Fixed.